clients/lab-sensible-terminal

```diff
@@ -0,0 +1,50 @@
+#!/bin/sh
+
+terminals="\
+	x-terminal-emulator \
+	alacritty \
+	foot \
+	kitty \
+	xfce4-terminal \
+	lxterminal \
+	qterminal \
+	mate-terminal \
+	gnome-terminal \
+	konsole \
+	sakura \
+	dtterm \
+	kgx \
+	nxterm \
+	rxvt \
+	st \
+	terminator \
+	terminology \
+	tilix \
+	color-xterm \
+	urxvt \
+	termit \
+	Eterm \
+	aterm \
+	uxterm \
+	xterm \
+	roxterm \
+	termite \
+	lilyterm \
+	terminix \
+	guake \
+	tilda \
+	hyper \
+	wezterm \
+	rio"
+
+for terminal in $terminals
+do
+	if type "${terminal}" >/dev/null 2>&1
```

Comment:
Mostly curiosity. Why `type` instead of `command -v`? I see the latter more in checks if a binary exists.

Reply:
Because that's what git use.

https://github.com/git/git/blob/cf6f63ea6bf35173e02e18bdc6a4ba41288acff9/Documentation/CodingGuidelines#L87

Reply:
fyi:
https://pubs.opengroup.org/onlinepubs/9699919799/utilities/V3_chap02.html

lists `command` (but not `type`) as (builtin) commands. that also lists `hash` which
some has used to check if command exists -- just that `ksh` returns `0` even if it did
not find the command which path it wanted to remember...